vspec/model/vsstree.py

```diff
@@ -118,6 +119,9 @@ def __init__(self, name, source_dict: dict, parent=None, children=None, break_on
             
         if "comment" in source_dict.keys():
             self.comment = source_dict["comment"]
+        
+        if "instantiate" in source_dict.keys():
```

Comment:
After rebase I do not think that the two lines here are needed as they are handled by the generic mechanism introduced by Sebastian. But I am thinking if it would make sense to have a check that verifies that the value after "instantiate" actually is a boolean value. Today you will not get an error if you use `instantiate: Falsch`, I think it will be treated as  `instantiate: True`

If we do not want to allow the flexible Python (or is yaml?) approach to allow a lot of synonyms like `On` and `Off`, we should maybe check that it is exactly `true`or `false` (doc PR currently use only lower case letters)

Reply:
true, will fix that